src/frontend/src/pages/myfeeds.js

```diff
@@ -37,72 +95,108 @@ export default function MyFeeds() {
           'Content-Type': 'application/json',
         },
         body: JSON.stringify({
-          author: feedAuthor,
-          url: feedUrl,
+          user: userInfo.id,
+          author: newFeedAuthor,
+          url: newFeedUrl,
         }),
       });
-      console.log(response);
-      return response;
+      setSubmitStatus(
+        response.ok
+          ? { message: 'Feed added successfully', isError: false }
+          : { message: `Error: ${response.status} ${response.statusText}`, isError: true }
+      );
     } catch (error) {
-      console.log({ error });
-      return { error };
+      setSubmitStatus({ message: error.message, isError: true });
+      console.log('Error adding feeds', error);
+    }
+  }
+
+  function handleChange(event) {
+    const { name, value } = event.target;
+    if (name === 'author') {
+      setNewFeedAuthor(value);
+    } else {
+      setNewFeedUrl(value);
```

Comment:
@humphd I'm surprised this part works, as I thought we couldn't do conditionals with hooks.